.azure/templates/build.yml

```diff
@@ -98,26 +150,35 @@ jobs:
       configuration: release
       msbuildArgs: -m
 
-  - task: CopyFiles@2
-    displayName: Filter Artifacts
-    condition: and(succeeded(), eq('${{ parameters.publish }}', 'true'))
-    inputs:
-      sourceFolder: artifacts/bin
-      contents: '**/!(*.ilk|*.exp|*.lastcodeanalysissucceeded)'
-      targetFolder: $(Build.ArtifactStagingDirectory)/bin
-
-  - task: PublishBuildArtifacts@1
-    displayName: Upload Artifacts
-    condition: and(succeeded(), eq('${{ parameters.publish }}', 'true'))
-    inputs:
-      artifactName: artifacts
-      pathToPublish: $(Build.ArtifactStagingDirectory)
-      parallel: true
-
-  - task: PublishSymbols@2
-    condition: and(succeeded(), eq('${{ parameters.publish }}', 'true'))
-    inputs:
-        sourceFolder: $(Build.ArtifactStagingDirectory)/bin
-        searchPattern: '**/*.pdb'
-        symbolServerType: teamServices
-        indexSources: false
+  - ${{ if eq(parameters.onebranch, true) }}:
+    # https://eng.ms/docs/cloud-ai-platform/azure-edge-platform-aep/aep-engineering-systems/productivity-and-experiences/onebranch-platform-services/onebranch/signing
+    - task: onebranch.pipeline.signing@1
+      displayName: Sign Installer
+      inputs:
+        command: 'sign'
+        signing_profile: 'external_distribution'
+        files_to_sign: '**/*.msi' # Only supports usermode binaries
```

Comment:
There's not much point in test signing the MSI if the drivers aren't signed, right?

Reply:
Well, it makes it easier to download without Edge/Chrome complaining.

Reply:
Ok let's see what happens. It'll need to be re-signed after the drivers get manually prod signed, which IDK how we'd do in the pipeline.